packages/eslint-plugin/typings/eslint-rules.d.ts

```diff
@@ -411,15 +411,13 @@ declare module 'eslint/lib/rules/no-unused-expressions' {
 
   const rule: TSESLint.RuleModule<
     'expected',
-    (
-      | 'all'
-      | 'local'
-      | {
-          allowShortCircuit?: boolean;
-          allowTernary?: boolean;
-          allowTaggedTemplates?: boolean;
-        }
-    )[],
+    [
+      {
+        allowShortCircuit?: boolean;
+        allowTernary?: boolean;
+        allowTaggedTemplates?: boolean;
+      }?,
+    ],
```

Comment:
I think it was wrong typed since there is no option "all", "local" in [eslint no-unused-expression](https://eslint.org/docs/rules/no-unused-expressions#options)